packages/plugin-essentials/sources/commands/exec.ts

```diff
@@ -31,15 +35,13 @@ export default class ExecCommand extends BaseCommand {
     const {project, locator} = await Project.find(configuration, this.context.cwd);
 
     return await xfs.mktempPromise(async binFolder => {
-      const {code} = await execUtils.pipevp(this.commandName, this.args, {
+      return await execute([this.commandName, ...this.args].join(` `), [], {
```

Comment:
The args can't be joined because commands such as `yarn exec node -p 'process.argv' 'a b'` will work incorrectly.

```suggestion
      return await execute(this.commandName, this.args, {
```

Reply:
Well that was a major oversight on my part, thanks 👍